mesh/activation.go

```diff
@@ -6,12 +6,17 @@ import (
 	"github.com/davecgh/go-xdr/xdr2"
 	"github.com/spacemeshos/go-spacemesh/common"
 	"github.com/spacemeshos/go-spacemesh/crypto"
+	"github.com/spacemeshos/go-spacemesh/database"
+	"github.com/spacemeshos/go-spacemesh/log"
 	"github.com/spacemeshos/go-spacemesh/nipst"
+	"github.com/spacemeshos/go-spacemesh/rand"
 )
 
 //todo: choose which type is VRF
 type Vrf string
 
+const AtxProtocol = "AtxGossip"
+
```

Comment:
why is this here?, i think mesh should be kept separated from anything related to network i.e broadcasting listening ...